nise/generators/azure/azure_generator.py

```diff
@@ -297,9 +298,9 @@ def _get_location(self):
 
     def _add_common_usage_info(self, row, start, end, **kwargs):
         """Add common usage information."""
+        row["SubscriptionGuid"] = self.subscription_guid
         if self.azure_columns == AZURE_COLUMNS_V2:
             usage_account = choice(self.usage_accounts)
-            row["SubscriptionId"] = self.subscription_guid
             row["SubscriptionName"] = self.account_info.get("subscription_name")
```

Comment:
I don't really understand this change, the Azure docs don't indicate there is a field called `SubscriptionGuid`, just `SubscriptionId`: https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields

Reply:
AZURE LIE